install/kubernetes/ingress-sni-zvpn.yaml

```diff
@@ -0,0 +1,335 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: envoy-config
+  namespace: istio-system
+data:
+  envoy.yaml: |
+    stats_config:
+      use_all_default_tags: false
+
+    admin:
+      access_log_path: /dev/stdout
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 15000 }
+
+    dynamic_resources:
+      lds_config:
+        api_config_source:
+          api_type: REST_LEGACY
+          refresh_delay: 10s
+          cluster_name: [rds]
```

Comment:
rds?

Reply:
Funny - current LDS v1 hardcodes the cluster name for discovery as rds

I could define 2 clusters, pointing to same pilot - but once we use LDSv2 it won't be a problem, and I can use [xds] name.